frontend/src/pages/DashboardPage/ChartComponenet/ChartComponent.jsx

```diff
@@ -145,7 +145,7 @@ const ChartComponent = forwardRef(({ chartData, sliderValue }, ref) => {
               label: (tooltipItem) => {
                 const item = accuracyData[tooltipItem.dataIndex];
                 return [
-                  `Accuracy: ${item.accuracy}`,
+                  `Bias: ${1 - item.accuracy}`,
```

Comment:
We should probably add this in the info button?

Reply:
Thats a good catch. We need to work on that